src/Evolution/Initialization/DgDomain.hpp

```diff
@@ -61,11 +61,13 @@ namespace evolution::dg::Initialization {
 /// \details See the type aliases defined below for what items are added to the
 /// GlobalCache and DataBox and how they are initialized
 
-template <size_t Dim, bool UseControlSystems = false>
+template <typename Metavariables, bool UseControlSystems = false>
 struct Domain {
+  static constexpr size_t dim = Metavariables::volume_dim;
+
```

Comment:
[optional] Could make it `Dim` and then nothing in the rest of the class would have to change.

Reply:
I could, but that would violate our style guide.